ZarCare Automation/Datasets/Register.json

```diff
@@ -1,8 +1,96 @@
 ﻿{
-  "First_Name": "Adams",
-  "Sur_Name": "Zampa",
-  "CellPhone_Number": "350451552",
-  "Email_Address": "adamzampa24092223@gmail.com",
-  "Pass_Word": "Test@2405",
-  "Confirm_Password": "Test@2405"
-}
\ No newline at end of file
+
+  "Valid_PatientDetail": [
+    {
+      "First_Name": "Adams"
+    },
+    {
+      "Sur_Name": "Zampa"
+    },
+    {
+      "CellPhone_Number": "350451552"
+    },
+    {
+      "Email_Address": "adamzampa24092223@gmail.com"
+    },
+    {
+      "Pass_Word": "Test@2405"
+    },
+    {
+      "Confirm_Password": "Test@2405"
+    }
+  ],
+  "Invalid_Patient_Detail": [
+    {
+      "Dupicate_First_Name": "Adams"
+    },
+    {
+      "Duplicate_last_Name": "Zampa"
+    },
+    {
+      "Invalid_CellPhoneNumber": "35045155235"
+    },
+    {
+      "Invalid_Email_address": "Test@.com"
+    },
+    {
+      "Invalidpassword": "abcd1234"
+    },
+    {
+      "Invalid_confirm_password": "abcd1234"
+    },
+    {
+      "InvalidCellPhoneError": "Enter valid cellphone number"
+    },
+    {
+      "InvalidEmailError": "Enter valid email"
+    },
+    {
+      "InvalidPassError": "Please use a stronger password"
+    }
+  ],
+  "Validation_Message": [
+    {
+      "FnameReqValiMessage": "First Name is required"
+    },
+    {
+      "SurNameReqMessage": "Surname is required"
+    },
+    {
+      "ReqPassErrorMessage": "Password is required"
+    },
+    {
+      "ReqConfPassMessage": "Confirm password is required"
+    },
+    {
+      "TermsAndContionMessage": "Please accept terms and conditions"
+    }
+  ]
+}
+
+
+
+
+
+
+  //"First_Name": "Adams",
```

Comment:
Please remove the commented code from the branch

Reply:
Removed commented json value